README.md

```diff
@@ -132,7 +147,7 @@ Valid settings:
     - `append`: append the `.css` extension to the SASS / SCSS file name: `foo.scss` will be compiled to `foo.scss.css`.
     - `replace`: replace the SASS / SCSS file extension by `.css`: `foo.scss` will be compiled to `foo.css`.
 - `pngCrushPath`: default is `pngcrush`
-- `bidiCss` (boolean): default is `false` (note that this is `true` by default in the `scissors/css_mirroring_transformer`, see below)
+- `bidiCss` (`skip` or `flip` or `biDirectionalize`): default is `skip` (note that this is `biDirectionalize` by default in `scissors/css_mirroring_transformer`, see below)
```

Comment:
As discussed earlier, I'd keep bidiCss and deprecate it (i.e. would remove it from doc but keep it in code).
Flip doesn't produce bidirectional CSS (note: camel seems out of place here as this word actually exists).

I was thinking of some name like `cssjanus: mirrored, bidirectional, skip`, WDYT?